zebra/zebra_script.h

```diff
@@ -0,0 +1,45 @@
+/* Frrscript encoders and decoders for data structures in zebra.
+ *
+ * Copyright (C) 2021 Donald Lee
+ *
+ * This file is part of FRR.
+ *
+ * FRR is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2, or (at your option) any
+ * later version.
+ *
+ * FRR is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with FRR; see the file COPYING.  If not, write to the Free
+ * Software Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA
+ * 02111-1307, USA.
```

Comment:
Wrong header
http://docs.frrouting.org/projects/dev-guide/en/latest/workflow.html#source-file-header

Reply:
updated in 575a205